services/discourse/discourse.service.js

```diff
@@ -38,6 +39,10 @@ class DiscourseBase extends BaseJsonService {
 
 function DiscourseMetricIntegrationFactory({ metricName, property }) {
   return class DiscourseMetric extends DiscourseBase {
+    static get name() {
+      return camelcase(`Discourse ${metricName}`, { pascalCase: true })
```

Comment:
will `camelcase` remove the space in between `Discourse` and the value of `metricName`? if not I'm thinking we'd want ``Discourse${metricName}``

Reply:
`camelcase` removes spaces. An example from https://www.npmjs.com/package/camelcase:
```js
camelCase('foo bar');
//=> 'fooBar'
```

Reply:
Thanks!

Reply:
Also, `metricName` is lowercase, so the space causes us to get `DiscourseTopics` instead of `Discoursetopics`. I'll add a comment.